tests/soak-tests/src/test/java/org/apache/activemq/artemis/tests/soak/brokerConnection/mirror/ReplicatedBothNodesMirrorTest.java

```diff
@@ -185,19 +190,25 @@ private static void createMirroredServer(String serverName,
       brokerProperties.put("AMQPConnections." + connectionName + ".connectionElements.mirror.sync", "false");
       brokerProperties.put("largeMessageSync", "false");
 
-      brokerProperties.put("addressSettings.#.maxSizeMessages", "50");
-      brokerProperties.put("addressSettings.#.maxReadPageMessages", "2000");
-      brokerProperties.put("addressSettings.#.maxReadPageBytes", "-1");
-      brokerProperties.put("addressSettings.#.prefetchPageMessages", "500");
+      brokerProperties.put("mirrorAckManagerQueueAttempts", "5");
+      brokerProperties.put("mirrorAckManagerPageAttempts", "500000");
+      brokerProperties.put("mirrorAckManagerRetryDelay", "500");
```

Comment:
Same overall feedback the earlier PR [1]. This seems unexpected for the non-paging case, I don't understand it being this way (or exactly what it is going to do based on your reply) at present , and its only going to be more difficult to understand it later. It could do with a comment explaining why the non-paging test is using paging config and what it is going to do, or changing to case-specific config given there is already a boolean being used to toggle the two different cases anyway.

[1] https://github.com/apache/activemq-artemis/pull/5164#discussion_r1731305285

Reply:
these are removed now.. I'm flushing the MirrorTarget from the AckManager now.